bodhi/server/models.py

```diff
@@ -554,6 +555,54 @@ def get_pkg_pushers(self, branch, settings):
 
         return (committers, watchers), (committergroups, watchergroups)
 
+    def get_pkg_committers_from_pagure(self, settings):
+        """
+        Pull users who can commit and are watching a package in Pagure.
+
+        Return two two-tuples of lists:
+        * The first tuple is for usernames.  The second tuple is for groups.
+        * The first list of the tuple is for committers. The second is for
+          watchers.
+        """
+        pagure_url = settings.get(
+            'pagure_url', 'https://src.fedoraproject.org/pagure/')
```

Comment:
I just put up https://github.com/fedora-infra/bodhi/pull/1544 which should make it possible to just do ```config['pagure_url']```. Not sure what is the best way to coordinate the two PRs, so let's just say that whichever one gets merged last should be sure to fix this and add the new setting to the new ```bodhi.server.config.BodhiConfig._defaults``` dictionary.